lib/tasks/update_dfid_subscriber_lists.rake

```diff
@@ -0,0 +1,34 @@
+desc "Update DFID subscriber lists"
+task update_dfid_subscriber_lists: :environment do
+  # Update all subscriber lists subscribed to dfid_research_output document types
+  SubscriberList.where("links->'content_store_document_type'->>'any' LIKE '%dfid_research_output%'").each do |sl|
+    new_links = sl.links.merge(
+      content_store_document_type: {
+        any: (sl.links[:content_store_document_type][:any] - %w[dfid_research_output] + %w[research_for_development_output]),
+      },
+    )
+
+    sl.update!(links: new_links)
+  end
+
+  # Update all subscriber lists subscribed to dfid_research_output document types (no idea what makes these different to the links equivalent)
+  SubscriberList.where("tags->'content_store_document_type'->>'any' LIKE '%dfid_research_output%'").each do |sl|
```

Comment:
It is a bit baffling about this, there seems to be only 2 of these whereas 131 of the link variant. The link one is perplexing too given that they are not links (i.e. not a content_id) - there's probably a grim cleanup job waiting in the future.

Reply:
It's very baffling 🤷‍♂️ I think as we've discovered in previous work that our subscriber lists are well due a cleanup!